stake-pool/js/test/mocks.ts

```diff
@@ -132,6 +141,66 @@ export function mockTokenAccount(amount = 0) {
   };
 }
 
+export const stakeAccountData = {
+  program: 'stake',
+  parsed: {
+    type: 'delegated',
+    info: {
+      meta: {
+        rentExemptReserve: new BN(1),
+        lockup: {
+          epoch: 32,
+          unixTimestamp: 2,
+          custodian: new PublicKey(12),
+        },
+        authorized: {
+          staker: new PublicKey(12),
+          withdrawer: new PublicKey(12),
+        },
+      },
+      stake: {
+        delegation: {
+          voter: new PublicKey(
+            new BN('e4e37d6f2e80c0bb0f3da8a06304e57be5cda6efa2825b86780aa320d9784cf8', 'hex'),
```

Comment:
I wanted this to match the ```validatorListMock.voteAccountAddress``` that's why i have to put a complicated value here and changing value in  ```validatorListMock.voteAccountAddress``` causing error in layout tests deepequality checks as it is expecting some extra zeroes in public key. but if you want i can put it in the ```CONSTANTS``` object and then use it that way it will look much clean

Reply:
Ok great, that works